modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -16,6 +16,55 @@
 
 toc::[]
 
+== Infinite Scale 6.5.0 (Rolling)
+
+IMPORTANT: This is a Rolling Release. Please check the {release-types-url}[documentation] to see if this release type is right for your use case.
+
+[discrete]
+=== Public and Private Links in the Sharing Panel
+
+The sharing panel has been redesigned to separate links into two categories: Permanent Link and Public Links.
+
+- *Permanent Link*: A fixed link for internal use that works as a pointer for people with existing access. Use it ex. to remind someone to review a file. The link remains unchanged, even if the file or folder is renamed or moved.
```

Comment:
```suggestion
- *Permanent Link*: A fixed link for internal use that works as a pointer for people with existing access. Use it e.g. to remind someone to review a file. The link remains unchanged, even if the file or folder is renamed or moved.
```

Reply:
Could you also add something like `The permanent link is located by the space memberships and people shares.`